webpack/ForemanTasks/Components/Chart/Chart.js

```diff
@@ -0,0 +1,128 @@
+/*
+  Copied from react-c3js: https://github.com/bcbcarl/react-c3js/blob/master/src/index.js
+  Added missing features:
+    * onChartCreate
+    * onChartUpdate
+
+  TODO: update react-c3js or move to patternfly-react
```

Comment:
I'll go ahead and add a task for this, would be good to get these changes into react-c3js.